### ISSUE TITLE
Size of downloaded offline regions

### ISSUE THREAD

**Issue by noorbakerally** [m0nac0/flutter-mapbox-gl#223]
Is it possible to get metadata about the downloaded region such as the date the download took place and the disk size taken up by the region on disk?

**Comment by mariusvn** [comment]
We still need to work on the download.
The priority is to allow more control over the ambiant cache.

**Comment by noorbakerally** [comment]
@mariusvn ok, no worries, thanks

**Comment by noorbakerally** [comment]
@mariusvn What is the size of the tile downloaded, is it 256x256px or 512x512px, for the region, I can calculate the number of tiles, so if i know the size of one standard tile, i can multiply and get an approximate size

**Comment by mariusvn** [comment]
[here are for 256x256](https://toolstud.io/photo/filesize.php?imagewidth=256&imageheight=256) for png, should be 262.14KB
and [here are for 512x512](https://toolstud.io/photo/filesize.php?imagewidth=512&imageheight=512) for png shoud be 1.05MB

but there are lossless png compression sometimes so, it can change quite a lot. The numbers up there are for uncompressed pngs

**Comment by noorbakerally** [comment]
@mariusvn no, my question was, do the plugin download 256x256px or 512x512px tile?

**Comment by mariusvn** [comment]
depends on your source settings, it can do both. I think the standard one is 256
